ros/processor/insights_engine_result_consumer.py

```diff
@@ -114,7 +126,28 @@ def process_report(self, host, reports):
                     fqdn=host['fqdn'],
                     rule_hit_details=reports,
                     number_of_recommendations=rec_count,
-                    state=SYSTEM_STATES[state_key]
+                    state=SYSTEM_STATES[state_key],
+                    instance_type=performance_record.get('instance_type')
+                )
+                LOG.info(
+                    f"{self.prefix} - System created/updated successfully: {host['id']}"
+                )
+
+                performance_utilization = {
+                    'memory': system_details['mem_utilization'],
+                    'cpu': system_details['cpu_utilization'],
+                    'io': system_details['io_utilization']
```

Comment:
@saltgen Good choice on keeping `system_details` consistent with `system_metadata`. But, would it make more sense either to keep `system_metadata` as is or rename it to `performance_details` or `performance_*`. The reason being if we keep it `system_details`, it should not be misinterpreted as any field or details of our `systems` table in ROS. Let me know what you think. Example added as a suggestion below.
```suggestion
                performance_utilization = {
                    'memory': system_metadata['mem_utilization'],
                    'cpu': system_metadata['cpu_utilization'],
                    'io': system_metadata['io_utilization']
```

Reply:
Thank you!
Yeah makes sense, updated `system_metadata` to `utilization_info`
Changes pushed :+1: